build.gradle.kts

```diff
@@ -42,3 +46,23 @@ spotless {
         ktlint("0.50.0")
     }
 }
+
+fun Project.configureSentry() {
+    extensions.getByType(SentryPluginExtension::class.java).apply {
+        includeProguardMapping = System.getenv()["CI"].toBoolean()
+                && !project.properties["skipSentryProguardMappingUpload"]?.toString().toBoolean()
+
+        tracingInstrumentation {
+            features.set(EnumSet.allOf(InstrumentationFeature::class.java) - InstrumentationFeature.FILE_IO)
+        }
+        autoInstallation.enabled = false
+        includeDependenciesReport = false
+        ignoredBuildTypes = setOf("debug")
```

Comment:
We have 3 build variants.

* `debug` - Debug builds that communicate with staging.
* `debugProd` - Debug builds that communicate with production.
* `release` - Release builds.

Shouldn't this also include `debugProd` variant?

Reply:
Good point, thanks! Addressed in c82eb3c